runtime/defaults/src/main/resources/application.properties

```diff
@@ -126,6 +126,10 @@ polaris.secrets-manager.type=in-memory
 polaris.file-io.type=default
 
 polaris.event-listener.type=no-op
+# polaris.event-listener.type=aws-cloudwatch
+# polaris.event-listener.aws-cloudwatch.log-group=test-group
+# polaris.event-listener.aws-cloudwatch.log-stream=test-stream
+# polaris.event-listener.aws-cloudwatch.region=us-west-2
```

Comment:
The intent behind these comments is good, but since it's a bit cloud-specific we might be better off leaving it out

Reply:
Sure, that makes sense. Where do you recommend we document this instead?

Reply:
Maybe if we had an application.properties specific to AWS, such as in the getting started section?